blackbox/docs/configuration.txt

```diff
@@ -1899,6 +1912,80 @@ Javascript Language
   Setting to enable the Javascript language. As The Javascript language is an
   experimental feature and is not securely sandboxed its disabled by default.
 
+Authentication
+==============
+
+Authentication settings are node settings, which means that their values apply
```

Comment:
@dobe hm, so `auth.host_based.enabled` is a cluster setting?
I understood that wrong then yesterday.